model/change_point_detection.go

```diff
@@ -369,10 +369,21 @@ func ReplaceChangePoints(ctx context.Context, env cedar.Environment, performance
 	return catcher.Resolve()
 }
 
-func GetTotalPagesForChangePointsGroupedByVersion(ctx context.Context, env cedar.Environment, projectId string, pageSize int) (int, error) {
-	countKey := "count"
-	pipe := appendAfterBaseGetChangePointsByVersionAgg(projectId, bson.M{
-		"$count": countKey,
+type GetChangePointsGroupedByVersionOpts struct {
```

Comment:
If Opts are usually persisted, should we perhaps just do Opts/Args. IMO this would make clear that these are just arguments for an API.

Reply:
If we're persisting this structure to the db (which I don't follow, because it seems just like an argument structure passed between different internal APIs, but maybe I'm reading it wrong) then it needs bson tags, and the like

Reply:
No I mean: we're not persisting to DB -- would renaming Opts-->Args so that its clear this is just a struct to pass arguments, rather than some kind of options on a domain model?

Reply:
I think if you name it "Options" its clear enough. I think having a panic'ing marshaler is a good way to ensure it doesn't get stuffed in the DB, but that's optional.